engine/datapool-collector/src/main/kotlin/io/holunda/camunda/datapool/sender/SimpleDataEntryCommandSender.kt

```diff
@@ -75,10 +78,35 @@ class SimpleDataEntryCommandSender(
 
   override fun sendDataEntryCommand(command: CreateOrUpdateDataEntryCommand) {
     if (properties.enabled) {
-      gateway.send<Any, Any?>(command) { m, r -> logger.debug("Successfully submitted command $m, $r") }
+      gateway.send<Any, Any?>(command) { m, r ->
+        if (r.isExceptional) {
+          errorHandler.apply(m, r)
+        } else {
+          successHandler.apply(m, r)
+        }
+      }
     } else {
       logger.debug("Would have sent command $command")
     }
   }
 }
 
+/**
+ * Success handler, logging.
+ */
+class LoggingCommandSuccessHandler(private val logger: Logger) : DataEntryCommandSuccessHandler {
+
+  override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+    logger.debug("Successfully submitted command $commandMessage, $commandResultMessage")
+  }
+}
+
+/**
+ * Error handler, logging the error.
+ */
+class LoggingCommandErrorHandler(private val logger: Logger) : DataEntryCommandErrorHandler {
+
+  override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+    logger.error("SENDER-006: Sending command $commandMessage resulted in error ${commandResultMessage.exceptionResult()}")
```

Comment:
If we have an exception here, shouldn't we provide it to the logger so it prints the stacktrace?